src/main/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/PostgresDirectoryDAO.scala

```diff
@@ -600,6 +604,10 @@ class PostgresDirectoryDAO(protected val writeDbRef: DbReference, protected val
     })
   }
 
+  override def disableAllIdentities(samRequestContext: SamRequestContext): IO[Unit] = {
+    IO.unit //no-op for now. throw exception maybe?
```

Comment:
I don't have a strong opinion on a no-op vs. throwing an exception here and in `createEnabledUsersGroup`. no-op seems fine to me